### ISSUE TITLE
.NetworkSettings.IPAddress does not work for non-default Docker networks

### ISSUE THREAD

**Issue by JeanMertz** [containous/traefik#226]
```toml
[docker]
endpoint = "unix:///var/run/docker.sock"
domain = "docker.localhost"
watch = true

[web]
address = ":8080"
```

```
docker run -d -p 8080:8080 -p 80:80 -v /var/run/docker.sock:/var/run/docker.sock -v $PWD/traefik.toml:/traefik.toml containous/traefik
docker run swarm:1.0.0 manage token://blablabla
```

![screen shot 2016-02-27 at 00 30 37](https://cloud.githubusercontent.com/assets/383250/13368294/6d86b35a-dce9-11e5-8891-e2694e7d8357.png)

```
docker network create bla
docker run -d --net=bla -p 8080:8080 -p 80:80 -v /var/run/docker.sock:/var/run/docker.sock -v $PWD/traefik.toml:/traefik.toml containous/traefik
docker run --net=bla swarm:1.0.0 manage token://blablabla
```

![screen shot 2016-02-27 at 00 30 18](https://cloud.githubusercontent.com/assets/383250/13368325/b23a8c56-dce9-11e5-921f-bc3406f675de.png)

This especially hurts docker-compose setups, as it creates a custom network by default.

**Comment by JeanMertz** [comment]
Turns out this is due to `{{.NetworkSettings.IPAddress}}` being empty, and [is deprecated](https://github.com/docker/docker/issues/18563#issuecomment-163603765).

There probably needs to be a setting to configure which network you want to use per container, but a quick workaround is to override the default template with:

```
{{range $i := .NetworkSettings.Networks}}{{if $i}}{{.IPAddress}}{{end}}{{end}}
```

However, it seems the currently stable `containous/traefik` Docker image probably contains an older version of the [github.com/fsouza/go-dockerclient](https://github.com/fsouza/go-dockerclient) library, as it doesn't recognise `Networks` as a valid field:

```
docker.tmpl:3:61: executing \"docker.tmpl\" at <.NetworkSettings.Net...>: Networks is not a field of struct type *docker.NetworkSettings"
```

From the looks of it, this feature was merged in [november last year](https://github.com/fsouza/go-dockerclient/pull/417).

**Comment by emilevauge** [comment]
Thanks a lot @JeanMertz for reporting and analyzing this :)
I just created a PR that fixes this #227. Can you test using this branch?
